src/angular-public/autocomplete/autocomplete/autocomplete.component.ts

```diff
@@ -97,6 +97,14 @@ export class AutocompleteComponent implements AfterContentInit, HasOptions {
   /** Function that maps an option's control value to its display value in the trigger. */
   @Input() displayWith: ((value: any) => string) | null = null;
 
+  /**
+   * Function which normalizes input values to highlight them in options.
+   * E.g. If your function is <code>(value: string) => value.replace(new RegExp('[ö]', 'i'), 'o')</code>
+   * and you search for 'Faroer', an option like 'Faröer' will be highlighted.
+   * IMPORTANT: your function must not change the length of the string.
```

Comment:
```suggestion
   * IMPORTANT: The provided function MAY NOT change the order of the characters or the length of the string.
   * (e.g. changing `ä` to `ae` would break the highlighting function)
```